test/user.test.js

```diff
@@ -205,6 +205,73 @@ describe('User', function() {
       });
     });
 
+    describe('Password validation with Maxlength', function() {
+      var pass72 = 'ThePasswordShouldNotExceedtheMaximumNumberOfcharactersAndItisInvalidIfIt';
+      var pass73 = 'ThePasswordShouldNotExceedtheMaximumNumberOfcharactersAndItisInvalidIfIt1';
+      var badPass = 'ThePasswordShouldNotExceedtheMaximumNumberOfcharactersAndItisInvalidIfItXYZ1';
+
+      it('rejects passwords longer than 72 characters', function(done) {
+        try {
+          User.create({ email: 'b@c.com', password: pass73 }, function(err) {
+            if (err) return done(err);
+            assert.equal(err.statusCode, 422);
+            assert(false, 'Error should have been thrown');
+          });
+        } catch (e) {
+          done();
+        }
```

Comment:
+1

If you want to assert that `User.create` throws, which I find rather weird, to be honest, because `User.create` should report errors via callback, then use [should#throws](https://shouldjs.github.io/#should-throws). Also verify that an expected error was thrown, and not something like `TypeError: undefined is not a function`.